tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3329 @@
+      properties:
+        type:
+          type: integer
+          description: Set to value 3 to denote an NFT Unlock.
+        reference:
+          type: integer
+          description: Represents the index of a previous unlock.
+      required:
+        - type
+        - reference
+
+    MilestonePayload:
+      description: The Milestone Payload to be embedded into a block.
+      properties:
+        type:
+          type: integer
+          description: Set to value 7 to denote a Milestone Payload.
+        index:
+          type: integer
+          description: The index of the milestone.
+        timestamp:
+          type: integer
+          description: The Unix timestamp at which the milestone was issued. The unix timestamp is specified in seconds.
+        protocolVersion:
+          type: integer
+          description: Protocol version of the Milestone Payload and its encapsulating block.
+        previousMilestoneId:
+          type: string
+          description: The Milestone ID of the milestone with Index Number - 1.
+        parents:
+          description: The identifiers of the blocks this milestone references. Hex-encoded values with 0x prefix.
+          type: array
+          items:
+            type: string
+        inclusionMerkleRoot:
+          type: string
+          description: The merkle root of all directly/indirectly referenced blocks (their IDs) which are newly confirmed by this milestone. Hex-encoded with 0x prefix.
+        appliedMerkleRoot:
+          type: string
+          description: The merkle root of all blocks (their IDs) carrying ledger state mutating transactions. Hex-encoded with 0x prefix.
+        options:
+          type: array
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/ReceiptPayload'
+              - $ref: '#/components/schemas/ProtocolParamsMilestoneOpt'
+        metadata:
+          type: string
+          description: Hex-encoded binary data with 0x prefix.
+        signatures:
+          type: array
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/Ed25519Signature'
+          description: An array of signatures signing the serialized Milestone Essence. Hex-encoded with 0x prefix.
+      required:
+        - type
+        - index
+        - timestamp
+        - protocolVersion
+        - previousMilestoneId
+        - parents
+        - inclusionMerkleRoot
+        - appliedMerkleRoot
+        - signatures
+
+    TaggedDataPayload:
+      description: The Tagged Data Payload to be embedded into a block.
+      properties:
+        type:
+          type: integer
+          description: Set to value 5 to denote a Tagged Data Payload.
+        tag:
+          type: string
+          description: The indexation key to allow external tools to find/look up this block. It has a size between 1 and 64 bytes and must be encoded as a hex-string with 0x prefix.
```

Comment:
Should be `indexation tag` if it should be aligned with the tagFeature, but since it's not indexed by the node, maybe `indexation` should also be removed?

Reply:
fixed in https://github.com/iotaledger/tips/pull/57/commits/23a9f423480290a96c63794f356c9e29f42544e8